core/Cargo.toml

```diff
@@ -35,7 +35,7 @@ termcolor = { version = "1", optional = true }
 toml = { version = "0.5", optional = true }
 tracing = { version = "0.1", optional = true }
 tracing-log = { version = "0.1", optional = true }
-tracing-subscriber = { version = "0.1", optional = true }
+tracing-subscriber = { version = "0.2", optional = true, default-features = false, features = ["fmt", "env-filter", "ansi", "smallvec", "tracing-log"] }
```

Comment:
As a note: the `smallvec` dependency is optional, it's used internally to tracing for performance optimizations. If you would prefer to not depend on `smallvec`, we can remove the feature flag that disables it; however, I suspect there may be a noticeable performance impact, so it may be worth benchmarking with and without.